src/Core/Grid/Definition/Factory/DeleteActionTrait.php

```diff
@@ -36,21 +36,35 @@
  */
 trait DeleteActionTrait
 {
-    protected function buildDeleteAction(string $deleteRouteName, string $deleteRouteParamName, string $deleteRouteParamField, string $method = 'POST'): RowActionInterface
-    {
+    protected function buildDeleteAction(
+        string $deleteRouteName,
+        string $deleteRouteParamName,
+        string $deleteRouteParamField,
+        string $method = 'POST',
+        array $extraRouteParams = [],
+        array $translations = []
```

Comment:
I'm not sure it is very useful to add this `$translations` parameter
Because it's not used anywhere, so it adds complexity for no reason Not very SOLID

Reply:
you are right, its no longer relevant since the wording have been made uniform